shell/platform/darwin/ios/framework/Source/accessibility_bridge.mm

```diff
@@ -200,8 +200,8 @@ void PostAccessibilityNotification(UIAccessibilityNotifications notification,
   for (SemanticsObject* object in [objects_ allValues]) {
     [object accessibilityBridgeDidFinishUpdate];
   }
-
-  if (!ios_delegate_->IsFlutterViewControllerPresentingModalViewController(view_controller_)) {
+  if (!ios_delegate_->IsFlutterViewControllerPresentingModalViewController(view_controller_) &&
+      [UIApplication sharedApplication].applicationState == UIApplicationStateActive) {
```

Comment:
I had a typo here that made it "worked". Looks like with a popup on the screen, the state is still active, so `[UIApplication sharedApplication].applicationState` should be used as an indicator here. Will need to find another way to do it. Setting this PR to draft until I fix it